docs/agents/distributed-tracing.md

```diff
@@ -0,0 +1,63 @@
+# Distributed Tracing
+
+We implement the W3C standards, both for HTTP headers and binary fields.
+
+To test compatibility with the standard,
+agents MUST validate against the w3c [test harness](https://github.com/w3c/trace-context/blob/master/test/test_data.json).
+
+## Http Headers
+
+Our implementation relies on the [W3C Trace Context](https://www.w3.org/TR/trace-context-1/) standard. Until this standard became final,
+we used the header name `elastic-apm-traceparent` and we did not support `tracestate`. Soon after the standard became official, we
+started to fully align with it. For backward compatibility reasons, this was done in phases, so that the first step was to look for both
+`traceparent` headers in incoming requests (meaning - both `elastic-apm-traceparent` and `traceparent`) and sending both `traceparent` headers in outgoing requests (with the exception of the RUM agent,
+due to CORS). [Issue #71](https://github.com/elastic/apm/issues/71) describes this in more detail, as well as tracking implementation
+in the different agents.
+New agents may decide whether to support both `traceparent` headers (so to be compatible with older agent versions) or only the formal W3C
+header.
+
+## Tracestate
+
+For our own `elastic` `tracestate` entry we will introduce a `key:value` formatted list of attributes.
```

Comment:
How do you feel about truncating `elastic` to `estc` or even `es`? Maybe it makes it too cryptic to warrant saving a few characters.

Reply:
I'd prefer to spell it out, it's not a significant difference. I seem to recall @jahtalab created a public PR/issue in a W3C repo saying we would use "elastic" too... not sure where that is now.

Reply:
From what I remember at the time we discussed the name, we settled on `es` but since there was no formal way to request an ID from the working group (I only asked on the slack channel), the process was stalled. Now, the process is to create a PR on this repo: https://github.com/w3c/trace-state-ids-registry. I think at this point we can choose either `elastic` or `es` since we're not in the registry yet.

Reply:
Seems like the trend is to use two letters for the vendor key, so I'd vote for `es`.

Reply:
OK, `es` sounds fine then. Updated.

Reply:
Let's create a PR on the w3c repo after this PR got approved and before merging it.